independent-publisher-connectors/ChuckNorris.io/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Daniel Laskewitz"
```

Comment:
@Laskewitz Can you please add `stackOwner` as well

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)